src/Application.php

```diff
@@ -55,13 +59,29 @@ public function __construct(
         $this->manifestLoader = $manifestLoader;
         $this->environment    = $environment;
     }
-
+    
+    /**
+     * @return array
+     */
+    public function getDeveloperEnvironments()
+    {
+        return $this->developerEnvironments;
+    }
+    
+    /**
+     * @param array $developerEnvironments
+     */
+    public function setDeveloperEnvironments(array $developerEnvironments = null)
```

Comment:
There shouldn't be a default value for the `$developerEnvironments` argument in my opinion. And especially it shouldn't be `null` because `isDevelopment()` will fail on a `null` `$this->developerEnvironments`.

Perhaps you can change that to accept either an array or a simple string (that's then being casted to an array).